base/system/format/format.c

```diff
@@ -474,12 +475,36 @@ int wmain(int argc, WCHAR *argv[])
                                &serialNumber, &maxComponent, &flags,
                                fileSystem, ARRAYSIZE(fileSystem)))
     {
-        K32LoadStringW(GetModuleHandle(NULL), STRING_NO_VOLUME, szMsg, ARRAYSIZE(szMsg));
-        PrintWin32Error(szMsg, GetLastError());
-        return -1;
+        if (GetLastError() == ERROR_UNRECOGNIZED_VOLUME)
+        {
+            swprintf(fileSystem, L"%s", L"RAW");
```

Comment:
It would be of good practice to use String Safe API calls ([RtlStringCchPrintfW](https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/ntstrsafe/nf-ntstrsafe-rtlstringcchprintfw) in this case) as you already do in `QueryDeviceInformation`.

Reply:
I've opted for non-safe API because I do not see any risk of stack overrun here. I changed it to `wcscpy` though, because no actual string formatting was done here anyway. I could also use a safe function here instead (and include necessary header for that) if preferred.